dev/com.ibm.ws.ejbcontainer.management.j2ee_fat/publish/servers/com.ibm.ws.ejbcontainer.management.j2ee.fat/server.xml

```diff
@@ -2,7 +2,7 @@
     <include location="../fatTestPorts.xml"/>
 
     <featureManager>
-        <feature>servlet-3.1</feature>
+        <feature>servlet-4.0</feature>
```

Comment:
We should not update the base `server.xml` here.  It should stay with the EE7 feature set since we skip EE7 in the `CheckpointRule`